game-backend/src/MainController.ts

```diff
@@ -109,10 +110,8 @@ export default class MainController {
   generateUnit(): Unit {
     const unit = createUnit({
       id: this.hyperIdInstance(),
-      position: {
-        x: Math.floor(Math.random() * 20),
-        y: Math.floor(Math.random() * 20),
-      },
+      health: 100,
```

Comment:
Please extract the default health either to a constant or don't pass it as it's already handled as part of `createUnit`.